tests/utils/exec/occli/cmd_client.go

```diff
@@ -64,7 +77,30 @@ func RunCMD(cmd string) (stdout string, stderr string, err error) {
 	return
 }
 
-func DummyGetKubeConfigFromEnv() {
+func GetKubeConfigFromEnv(usersValue string) (kubeconfig string, err error) {
```

Comment:
I would change that method to 

```suggestion
func LoginAndGetKubeConfig(user string, password string, kubeconfigFilepath string) error {
```